frontend/src/metabase/dashboard/actions/parameters.unit.spec.js

```diff
@@ -0,0 +1,31 @@
+import { removeParameter, REMOVE_PARAMETER } from "./parameters";
+
+describe("removeParameter", () => {
+  let dispatch;
+  let getState;
+  beforeEach(() => {
+    dispatch = jest.fn();
+    getState = () => ({
+      dashboard: {
+        dashboardId: 1,
+        dashboards: {
+          1: {
+            id: 1,
+            parameters: [{ id: 123 }, { id: 456 }],
+          },
+        },
+        parameterValues: {
+          123: null,
+          456: null,
+        },
+      },
+    });
+  });
+  it("should return the `parameterId` as `payload.id` (metabase#34680)", async () => {
```

Comment:
1. It should be issue id, not PR id. Apologies for my [misleading suggestion](https://github.com/metabase/metabase/pull/34680#discussion_r1360214312).
2. Nit: we always put new lines here

```suggestion

  it("should return the `parameterId` as `payload.id` (metabase#33826)", async () => {
```

Reply:
updated! thanks for the feedback. I'll take a note for the issue id things. Thanks for your guidance!